skins/laika/src/components/pages/donation_form/Payment.vue

```diff
@@ -32,7 +32,7 @@
 					:payment-types="paymentTypes"
 					:error="typeIsValid ? '' : $t('donation_form_payment_type_error')"
 					:title="$t('donation_form_payment_type_title')"
-					:disabled-payment-types="[]"
+					:disabled-payment-types="emailOnlyDisabledPaymentTypes"
```

Comment:
why not use `disabledPaymentTypes` here?

Reply:
It was to do with this from the linked commit:

>when navigating back to payment page (in email bucket): don't disable any payment option based on address type selection (implemented with feature toggle)

Basically in the B version we don't want the addressType selection to make paymentTypes disabled. A side effect of that was it was also not making Sofort disabled when a non 0 interval was selected. We can't use the computed `disabledPaymentTypes` as that also disables based on addressType, so a new one that only disables based on the interval selected was needed.